app/assets/javascripts/mobile.js

```diff
@@ -30,11 +25,13 @@ $(function () {
         $this.text($this.text().replace(/Show/, 'Hide'));
         instance.$navbar.removeClass('page-navigation-closed');
         instance.$navbar.addClass('page-navigation-open');
+        GOVUK.analytics.trackEvent("contentsClicked", "openGuideNav",{});
```

Comment:
Tiny nit: We should also have spaces between the parameters passed into `trackEvent`